<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QPushButton" name="start">
      <property name="text">
       <string>开始计时</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QTableWidget" name="tableWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>开始时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>结束时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>持续时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>内容</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QDateTimeEdit" name="begainTime">
         <property name="displayFormat">
          <string>yyyy-MM-dd hh:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="addTime">
         <property name="text">
          <string>add</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="type">
         <item>
          <property name="text">
           <string>life</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>study</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>sleep</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>work</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>eating</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>traffic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>internet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>read</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>sport</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>tv</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>housework</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>other</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateTimeEdit" name="endTime">
         <property name="displayFormat">
          <string>yyyy-MM-dd hh:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QPlainTextEdit" name="text"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="stop">
      <property name="text">
       <string>停止计时</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="timeLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTomato">
    <property name="title">
     <string>tomato</string>
    </property>
    <addaction name="actionTomato_manager"/>
   </widget>
   <addaction name="menuTomato"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTomato_manager">
   <property name="text">
    <string>tomato manager</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
